service/internal/releaseversion/release_version.go

```diff
@@ -38,7 +38,7 @@ func New(c Config) (*ReleaseVersion, error) {
 	return rv, nil
 }
 
-func (rv *ReleaseVersion) AppVersion(ctx context.Context, obj interface{}) (map[string]string, error) {
+func (rv *ReleaseVersion) Apps(ctx context.Context, obj interface{}) (map[string]ReleaseApp, error) {
```

Comment:
This is a bit weird right now. The naming is weird. To me `Apps` makes more sense because the `releaseversion` package already has the word package.

Reply:
Cool this name makes more sense since it returns a ReleaseApp now.